packages/generator-single-spa/src/vue/generator-single-spa-vue.js

```diff
@@ -78,9 +78,21 @@ module.exports = class SingleSpaVueGenerator extends Generator {
       }
     );
 
+    if (error) {
+      if (error.message) {
+        console.error("The process encountered an error: ", error.message);
+      } else {
+        console.error("The process encountered an error: ", error);
+      }
```

Comment:
Printing only `error.message` to the console will prevent the stacktrace or other information from being logged. Additionally, passing multiple arguments to console.error doesn't ensure a new line, which could make the output a bit less readable. The logged message also could be more clear about which process has failed

```suggestion
      console.error(`The vue cli process to create a new project failed`);
      console.error(error);
```